digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -293,11 +294,19 @@ protected TaskResult callExecutor(Path projectPath, String type, TaskRequest mer
                 .build();
 
         DefaultSecretProvider secretProvider = new DefaultSecretProvider(
-                secretContext, secretAccessPolicy, grants, operatorSecretFilter, secretStore);
+                secretContext, secretAccessPolicy, grants, (key) -> declaredSecretKeys.contains(key), secretStore);
+
+        PrivilegedVariables privilegedVariables = GrantedPrivilegedVariables.build(
+                mergedRequest.getConfig().getNestedOrGetEmpty("_env"),
```

Comment:
"_env" is set of environment variables given by workflow developers. Not by system administrators. commented more on https://github.com/treasure-data/digdag/pull/340#discussion_r82857853